.github/scripts/format-check

```diff
@@ -1,14 +1,12 @@
-#!/bin/bash -eux
+#!/bin/bash -ae
```

Comment:
Using `set -a` (equivalent to setting `-a` as an argument to `bash` as is done here) ensures that all variables and functions set thereafter are exported. This eliminates the need for a separate `export -f unformatted` statement.